elfloader-tool/src/arch-riscv/crt0.S

```diff
@@ -62,7 +62,7 @@ _start:
 hsm_switch_hart:
   li a7, SBI_HSM_BASE
   li a6, SBI_HSM_BASE_HART_START
-  li a0, 1       // hart id to start
+  li a0, CONFIG_FIRST_HART_ID       // hart id to start
```

Comment:
Please fix the comment `/* start hart 1*/` then also.

Reply:
@axel-h  fixed